tests/story.test.js

```diff
@@ -235,12 +236,12 @@ it('can find stories by placeID - findStoryByPlaceID API', async () => {
                 .expect(200, done);
         });
 
-        it('/stories/story/:storyId - return story by storyId', function (done) {
-            request(app).get('/stories/story/1')
-                .set('Accept', 'application/json')
-                .expect('Content-Type', /json/)
-                .expect(200, done);
-        });
+        // it('/stories/story/:storyId - return story by storyId', function (done) {
```

Comment:
We can also remove this test. We do have a test for findStoryByStoryID on L214. Since this test is not compatible with the new change of using the mongodb generated ID, we can remove the test. Code coverage will still be the same

Reply:
Removed.